CHANGELOG.md

```diff
@@ -10,11 +10,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Support for item.Attachment:Mail restore
+- Errors from duplicate names in Exchange Calendars
+
+### Breaking Change: Exchange Calendar Storage
+Exchange Calendars storage entries now construct theirs paths using container IDs instead of display names.  This fixes cases where duplicate display names caused system failures.  The migration conflicts with existing calendar backups, producing the following problems:
```

Comment:
```suggestion
Exchange Calendars storage entries now construct their paths using container IDs instead of display names.  This fixes cases where duplicate display names caused system failures.  The migration conflicts with existing calendar backups, producing the following problems:
```